pkg/api/defaults.go

```diff
@@ -774,7 +775,7 @@ func (p *Properties) setTelemetryProfileDefaults() {
 	}
 
 	if len(p.TelemetryProfile.ApplicationInsightsKey) == 0 {
```

Comment:
I wonder if we should rename this something like UserApplicationInsightsKey/SecondaryApplicationInsightsKey (or anything that would differential it from the one our team will use to collect data) for code clarity

Reply:
All of the settings in the api-model are for users. I think it makes sense for a user with the current name. Really, they should not need to know about the system key. (IMO)

Reply:
Agreed, but code maintainers should be able to easily distinguish.
Whatever makes that easiest.